lib/graph_ql/directives/include_directive.rb

```diff
@@ -1,15 +1,9 @@
 GraphQL::IncludeDirective = GraphQL::Directive.new do |d|
-  d.name "include"
-  d.description "Include this part of the query if `if` is true"
+  d.name 'include'
+  d.description 'Include this part of the query if `if` is true'
   d.on([GraphQL::Directive::ON_FIELD, GraphQL::Directive::ON_FRAGMENT])
-  d.arguments({
-    if: d.arg({type: !GraphQL::BOOLEAN_TYPE})
-  })
-  d.resolve -> (arguments, proc) {
-    if arguments["if"]
-      proc.call
-    else
-      nil
-    end
+  d.arguments(if: d.arg(type: !GraphQL::BOOLEAN_TYPE))
+  d.resolve lambda  { |arguments, proc|
+    proc.call if arguments['if']
```

Comment:
I'm not into line-final conditionals -- I prefer code whose left-to-right, top-to-bottom reading matches its execution as closely as possible!